src/modules/dashboard/components/Dialogs/RemoveSafeDialog/RemoveSafeDialogForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { FormikProps } from 'formik';
+
+import Button from '~core/Button';
+import DialogSection from '~core/Dialog/DialogSection';
+import Heading from '~core/Heading';
+
+import { useLoggedInUser, Colony } from '~data/index';
+import { useTransformer } from '~utils/hooks';
+import { getAllUserRoles } from '~modules/transformers';
+import { canEnterRecoveryMode } from '~modules/users/checks';
+
+import SafeListItem from './SafeListItem';
+import { Safe } from './types';
+import { FormValues } from './RemoveSafeDialog';
+
+import styles from './RemoveSafeDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.RemoveSafeDialog.RemoveSafeDialogForm.title',
+    defaultMessage: 'Remove safe',
```

Comment:
Although, not in the spec, I think that "safe" should be capitalised here to "Safe". Given it is a name.